plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/StoreConnect.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Task;
+
+/**
+ * Connect store to Woo.com Task
+ */
+class StoreConnect extends Task {
+	/**
+	 * ID.
+	 *
+	 * @return string
+	 */
+	public function get_id() {
+		return 'connect-store';
+	}
+
+	/**
+	 * Title.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		return __( 'Connect store to Woo.com', 'woocommerce' );
+	}
+
+	/**
+	 * Content.
+	 *
+	 * @return string
+	 */
+	public function get_content() {
+		return '';
+	}
+
+	/**
+	 * Time.
+	 *
+	 * @return string
+	 */
+	public function get_time() {
+		return '';
+	}
+
+	/**
+	 * Task completion.
+	 *
+	 * @return bool
+	 */
+	public function is_complete() {
+		return \WC_Helper::is_site_connected();
```

Comment:
Ideally, I think there should also be a mechanism for users to decline this so that the task can still be completed when the site doesn't need a Woo.com connection. Otherwise for a segment of users the task list can't be completed and continues to take up space on the home screen.

Reply:
Addressed [30f8b1e](https://github.com/woocommerce/woocommerce/pull/45795/commits/30f8b1ebff0ac8b251a449dd1a3ea0c793fa87f0)